connaisseur/validators/notrayv1/notary.py

```diff
@@ -60,68 +85,25 @@ def get_key(self, key_name: str = None):
         """
 
         try:
-            if len(self.pub_root_keys) < 2:
-                key = next(iter(self.pub_root_keys))["key"]
-            else:
-                key_name = key_name or "default"
-                key = next(
-                    key["key"] for key in self.pub_root_keys if key["name"] == key_name
-                )
+            key_name = key_name or "default"
+            key = next(
+                key["key"] for key in self.pub_root_keys if key["name"] == key_name
+            )
             return "".join(key)
```

Comment:
The `key_name` assignment can go before the `try` block, and the `return` statement can go after the `except` block